### DISCUSSION TITLE
Lesson 6: I cannot install foundry on my ubuntu computer.

### DISCUSSION THREAD

**Discussion by sharkz727** [Cyfrin/foundry-full-course-cu#279] [Q&A] [answered]
I have been trying to solve this issue for an hour using browser and AI and I don't know what the issue?

![vscode](https://github.com/Cyfrin/foundry-full-course-f23/assets/138640466/f31e26f0-f2f6-4f92-974f-6a88effbd94b)

**Comment by ChefAharoni** [comment, marked as answer]
Hi @sharkz727,

Please try the following [fixes](https://www.debugpoint.com/failed-connect-raw-githubusercontent-com-port-443/) for this problems.

**Comment by sharkz727** [comment]
Thanks @ChefAharoni now i solved it.

**Reply by ChefAharoni** [reply]
I'm glad to hear, please mark my solution as answer if it solved your problem.
